data/data.go

```diff
@@ -65,3 +65,71 @@ type StringModifiers struct {
 	I        bool `json:"i"` // for regex
 	S        bool `json:"s"` // for regex
 }
+
+type Keyword struct {
```

Comment:
Similar to `Operator`, it would probably be beneficial to do:

```go
type Keyword string

const (
    KeywordRule Keyword = "rule"
    ...
)
```

So that people don't have to deal with raw strings.